src/read_write/node_writer.rs

```diff
@@ -129,24 +147,7 @@ impl WriteLE for Color<u8> {
     }
 }
 
-impl WriteLE for Vec<f32> {
-    fn write_le(&self, writer: &mut DataWriter) -> Result<()> {
-        let mut bytes = vec![0; 4 * self.len()];
-        LittleEndian::write_f32_into(self, &mut bytes);
```

Comment:
@feuerste  Was this optimization important?

Reply:
I made this optimization, since the f32 implementation gets a reference, which is internally dereferenced, which is not optimal... and writing a vector of hundreds of thousands of points at once seems to be much faster.

Reply:
So you'd prefer we keep the two versions of the instance?

Reply:
Could we keep one for `cgmath::BaseNum`?

Reply:
How would I get the correct LittleEndian method? A macro?

Reply:
Note that I kept the Vector3 instances, this is only for Vec<_>

Reply:
Ok, then we may need a macro taking the type as str  and its number of bytes and call this for all our types. And we could have another one for the non `Vec` types, too, to reduce the amount of code, right?

Reply:
Done.